test/aktivitetskrav/testDataUtils.ts

```diff
@@ -56,11 +56,12 @@ export const createAktivitetskrav = (
 export const createAktivitetskravVurdering = (
   status: AktivitetskravStatus,
   arsaker: VurderingArsak[],
-  beskrivelse = ""
+  beskrivelse = "",
+  createdAt = new Date()
```

Comment:
`,`?

Reply:
Prettier ser ut til å fjerne dangling comma i funksjonsparametere hos meg. For objekt-properties f.eks. så blir de lagt på. Det er kanskje sånn det skal være? Eller har jeg annet prettier-oppsett?

Reply:
Nei, det må vel stemme det da, jeg regner med alle har samme oppsett? 🤔